pandas/tests/test_resample.py

```diff
@@ -41,7 +41,7 @@
 
 # The various methods we support
 downsample_methods = ['min', 'max', 'first', 'last', 'sum', 'mean', 'sem',
-                      'median', 'prod', 'var', 'ohlc']
```

Comment:
can you add the test from the OP which compares 2 methods of getting the same result